dsgrid/query/report_peak_load.py

```diff
@@ -34,23 +35,27 @@ def generate(
     ):
         if context.get_table_format_type() == TableFormatType.PIVOTED:
             value_columns = list(context.get_pivoted_columns())
+            group_by_columns = inputs.group_by_columns
         else:
-            # TODO #202: this is TBD
-            # value_columns = ["value"]
-            raise NotImplementedError(f"Unsupported {context.get_table_format_type()=}")
+            value_columns = [VALUE_COLUMN]
```

Comment:
Why not make this else as a check against UNPIVOTED instead? That way, there will never be more than one metric column...